components/carousel/index.jsx

```diff
@@ -78,7 +78,11 @@ class Carousel extends React.Component {
 			PropTypes.string,
 		]),
 		/**
-		 * Boolean showing whether the autoplay feature is available or not
+		 * Dictates the currently active/visible carousel panel. 1-indexed. Use with `onRequestPanelChange` for a controlled carousel component. If not provided, the carousel will manage this itself via state.
```

Comment:
I'm not sure I know of anything else 1-indexed in the library.

Reply:
Changed to 0 indexing & pushing soon